website/content/docs/secrets/databases/redis.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: Redis - Database - Secrets Engines
+description: |-
+Redis is one of the supported plugins for the database secrets engine.
+This plugin generates database credentials dynamically based on configured
+roles for the Redis database.
```

Comment:
This needs two spaces or it won't render correctly. Suggestion below should work.

```suggestion
  Redis is one of the supported plugins for the database secrets engine.
  This plugin generates database credentials dynamically based on configured
  roles for the Redis database.
```